thirdeye/thirdeye-frontend/app/pods/custom/anomalies-table/dimensions/template.hbs

```diff
@@ -1,14 +1,12 @@
 <ul class="te-anomaly-table__list">
-  <li class="te-anomaly-table__list-item te-anomaly-table__list-item--smaller" title="{{record.anomaly.functionName}}">
-    {{#link-to 'rootcause' (query-params anomalyId=record.id)}}
-      {{record.anomaly.functionName}}
-    {{/link-to}}
+  <li class="te-anomaly-table__list-item" title="{{record.anomaly.functionName}}">
+    <a href="{{record.alertQueryUrl}}">{{record.anomaly.functionName}}</a>
```

Comment:
Here, I'm pretty sure this will work, as we're doing it in self-serve tabs here: https://github.com/linkedin/pinot/blob/da30818ce7986e69d98aaf79103641f204704988/thirdeye/thirdeye-frontend/app/pods/manage/alert/template.hbs#L110

`{{#link-to "manage.alert.explore" record.anomaly.functionId}}`

Reply:
OK, i got what you're saying. Interesting that this pattern also works with this api. Will give it a try. Thanks Steve.